tests/EStimLibrary.UnitTests/Extensions/SpatialModel/StringHierarchy/StringHierarchySpecTests.cs

```diff
@@ -0,0 +1,552 @@
+using Xunit;
+using EStimLibrary.Extensions.SpatialModel.StringHierarchy;
+
+
+namespace EStimLibrary.UnitTests.Extensions.SpatialModel.StringHierarchy;
```

Comment:
I changed it here, but calling attention to it so you know in future: use filscoped namespace syntax, i.e., just put a ; at the end of the namespace line so you don't need the extra {} around the whole body of the file and don't need to indent that extra level.